res/skins/LateNight/samplers_unit_1-8_big.xml

```diff
@@ -0,0 +1,112 @@
+          <TooltipId>show_samplers</TooltipId>
+          <ObjectName>Samplerexpandbutton</ObjectName>
+          <Size>min,min</Size>
+          <NumberStates>2</NumberStates>
+          <State>
+            <Number>1</Number>
+            <Pressed>skin:/buttons/btn_samplerExpand.svg</Pressed>
+            <Unpressed>skin:/buttons/btn_samplerExpand.svg</Unpressed>
+          </State>
+          <State>
+            <Number>0</Number>
+            <Pressed>skin:/buttons/btn_samplerCollapse.svg</Pressed>
+            <Unpressed>skin:/buttons/btn_samplerCollapse.svg</Unpressed>
+          </State>
+          <Connection>
+            <ConfigKey>[Samplers],show_big</ConfigKey>
+            <ButtonState>LeftButton</ButtonState>
+          </Connection>
+      </PushButton>
+
+      <WidgetGroup><Size>2f,1min</Size></WidgetGroup>
+
+      <WidgetGroup>
+      <Layout>vertical</Layout>
+      <SizePolicy>min,min</SizePolicy>
+      <Children>
+
+        <!-- First row -->
+        <WidgetGroup>
+        <ObjectName>SamplerTextSmall</ObjectName>
+        <Layout>horizontal</Layout>
+        <SizePolicy>min,min</SizePolicy>
+        <Children>
+		    <Template src="skin:sampler.xml">
+		      <SetVariable name="samplernum">1</SetVariable>
+		    </Template>
+
+		    <WidgetGroup><Size>4f,1min</Size></WidgetGroup>
+
+		    <Template src="skin:sampler.xml">
+		      <SetVariable name="samplernum">2</SetVariable>
+		    </Template>
+
+		    <WidgetGroup><Size>12f,1min</Size></WidgetGroup>
+
+		    <Template src="skin:sampler.xml">
+		      <SetVariable name="samplernum">3</SetVariable>
+		    </Template>
+
+		    <WidgetGroup><Size>4f,1min</Size></WidgetGroup>
+
+		    <Template src="skin:sampler.xml">
+		      <SetVariable name="samplernum">4</SetVariable>
+		    </Template>
+        </Children>
+        </WidgetGroup>
+        <!-- /First row -->
+
+        <!-- Second row -->
+        <WidgetGroup>
+        <Layout>horizontal</Layout>
+        <Children>
+
+	        <Template src="skin:sampler.xml">
+	          <SetVariable name="samplernum">5</SetVariable>
+	        </Template>
+
+	        <WidgetGroup><Size>4f,1min</Size></WidgetGroup>
+
+	        <Template src="skin:sampler.xml">
+	          <SetVariable name="samplernum">6</SetVariable>
+	        </Template>
+
+	        <WidgetGroup><Size>12f,1min</Size></WidgetGroup>
+
+	        <Template src="skin:sampler.xml">
+	          <SetVariable name="samplernum">7</SetVariable>
+	        </Template>
+
+	        <WidgetGroup><Size>4f,1min</Size></WidgetGroup>
+
+	        <Template src="skin:sampler.xml">
+	          <SetVariable name="samplernum">8</SetVariable>
+	        </Template>
+
+	        </Children>
+	      </WidgetGroup>
+
+        </Children>
+      </WidgetGroup>
+
+      <WidgetGroup>
+        <Size>5f,1min</Size>
+        <Connection>
+          <ConfigKey>[Master],skin_settings</ConfigKey>
+          <BindProperty>visible</BindProperty>
+        </Connection>
+      </WidgetGroup>
```

Comment:
This spacer is already in samplers_rack.xml, it can be removed here

Reply:
ok, done